app/jobs/application_job.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 class ApplicationJob < ActiveJob::Base
   # Automatically retry jobs that encountered a deadlock
   # retry_on ActiveRecord::Deadlocked
```

Comment:
Purpose of this class?

Reply:
It appears that rails autogenerates this class and it is meant to implement parallel "jobs" in a rails application — something like billing or mailings. We don't use this now, but it would be useful in the future for implementing an announcements feature that sends emails to users.